lib/Resque/Redis.php

```diff
@@ -218,6 +217,20 @@ public static function parseDsn($dsn)
 			parse_str($parts['query'], $options);
 		}
 
+		//check 'password-hashing' parameter and extracting password based on encoding
```

Comment:
This should probably be called `password-encoding`, since we're not actually hashing anything at this point...

Reply:
Made the changes as suggested